### ISSUE TITLE
Unable to launch container on fresh jammy vm with freshly compiled lxd 5.0.0

### ISSUE THREAD

**Issue by dionysius** [lxc/lxd#10571]
# Required information

 * Distribution: ubuntu
 * Distribution version: jammy
 * The output of "lxc info":
 
 <details>
  <summary>Click to expand!</summary>

 ```
 root@lxd-by-source-debug-jammy:~# $(go env GOPATH)/bin/lxc info
config: {}
api_extensions:
- storage_zfs_remove_snapshots
- container_host_shutdown_timeout
- container_stop_priority
- container_syscall_filtering
- auth_pki
- container_last_used_at
- etag
- patch
- usb_devices
- https_allowed_credentials
- image_compression_algorithm
- directory_manipulation
- container_cpu_time
- storage_zfs_use_refquota
- storage_lvm_mount_options
- network
- profile_usedby
- container_push
- container_exec_recording
- certificate_update
- container_exec_signal_handling
- gpu_devices
- container_image_properties
- migration_progress
- id_map
- network_firewall_filtering
- network_routes
- storage
- file_delete
- file_append
- network_dhcp_expiry
- storage_lvm_vg_rename
- storage_lvm_thinpool_rename
- network_vlan
- image_create_aliases
- container_stateless_copy
- container_only_migration
- storage_zfs_clone_copy
- unix_device_rename
- storage_lvm_use_thinpool
- storage_rsync_bwlimit
- network_vxlan_interface
- storage_btrfs_mount_options
- entity_description
- image_force_refresh
- storage_lvm_lv_resizing
- id_map_base
- file_symlinks
- container_push_target
- network_vlan_physical
- storage_images_delete
- container_edit_metadata
- container_snapshot_stateful_migration
- storage_driver_ceph
- storage_ceph_user_name
- resource_limits
- storage_volatile_initial_source
- storage_ceph_force_osd_reuse
- storage_block_filesystem_btrfs
- resources
- kernel_limits
- storage_api_volume_rename
- macaroon_authentication
- network_sriov
- console
- restrict_devlxd
- migration_pre_copy
- infiniband
- maas_network
- devlxd_events
- proxy
- network_dhcp_gateway
- file_get_symlink
- network_leases
- unix_device_hotplug
- storage_api_local_volume_handling
- operation_description
- clustering
- event_lifecycle
- storage_api_remote_volume_handling
- nvidia_runtime
- container_mount_propagation
- container_backup
- devlxd_images
- container_local_cross_pool_handling
- proxy_unix
- proxy_udp
- clustering_join
- proxy_tcp_udp_multi_port_handling
- network_state
- proxy_unix_dac_properties
- container_protection_delete
- unix_priv_drop
- pprof_http
- proxy_haproxy_protocol
- network_hwaddr
- proxy_nat
- network_nat_order
- container_full
- candid_authentication
- backup_compression
- candid_config
- nvidia_runtime_config
- storage_api_volume_snapshots
- storage_unmapped
- projects
- candid_config_key
- network_vxlan_ttl
- container_incremental_copy
- usb_optional_vendorid
- snapshot_scheduling
- snapshot_schedule_aliases
- container_copy_project
- clustering_server_address
- clustering_image_replication
- container_protection_shift
- snapshot_expiry
- container_backup_override_pool
- snapshot_expiry_creation
- network_leases_location
- resources_cpu_socket
- resources_gpu
- resources_numa
- kernel_features
- id_map_current
- event_location
- storage_api_remote_volume_snapshots
- network_nat_address
- container_nic_routes
- rbac
- cluster_internal_copy
- seccomp_notify
- lxc_features
- container_nic_ipvlan
- network_vlan_sriov
- storage_cephfs
- container_nic_ipfilter
- resources_v2
- container_exec_user_group_cwd
- container_syscall_intercept
- container_disk_shift
- storage_shifted
- resources_infiniband
- daemon_storage
- instances
- image_types
- resources_disk_sata
- clustering_roles
- images_expiry
- resources_network_firmware
- backup_compression_algorithm
- ceph_data_pool_name
- container_syscall_intercept_mount
- compression_squashfs
- container_raw_mount
- container_nic_routed
- container_syscall_intercept_mount_fuse
- container_disk_ceph
- virtual-machines
- image_profiles
- clustering_architecture
- resources_disk_id
- storage_lvm_stripes
- vm_boot_priority
- unix_hotplug_devices
- api_filtering
- instance_nic_network
- clustering_sizing
- firewall_driver
- projects_limits
- container_syscall_intercept_hugetlbfs
- limits_hugepages
- container_nic_routed_gateway
- projects_restrictions
- custom_volume_snapshot_expiry
- volume_snapshot_scheduling
- trust_ca_certificates
- snapshot_disk_usage
- clustering_edit_roles
- container_nic_routed_host_address
- container_nic_ipvlan_gateway
- resources_usb_pci
- resources_cpu_threads_numa
- resources_cpu_core_die
- api_os
- container_nic_routed_host_table
- container_nic_ipvlan_host_table
- container_nic_ipvlan_mode
- resources_system
- images_push_relay
- network_dns_search
- container_nic_routed_limits
- instance_nic_bridged_vlan
- network_state_bond_bridge
- usedby_consistency
- custom_block_volumes
- clustering_failure_domains
- resources_gpu_mdev
- console_vga_type
- projects_limits_disk
- network_type_macvlan
- network_type_sriov
- container_syscall_intercept_bpf_devices
- network_type_ovn
- projects_networks
- projects_networks_restricted_uplinks
- custom_volume_backup
- backup_override_name
- storage_rsync_compression
- network_type_physical
- network_ovn_external_subnets
- network_ovn_nat
- network_ovn_external_routes_remove
- tpm_device_type
- storage_zfs_clone_copy_rebase
- gpu_mdev
- resources_pci_iommu
- resources_network_usb
- resources_disk_address
- network_physical_ovn_ingress_mode
- network_ovn_dhcp
- network_physical_routes_anycast
- projects_limits_instances
- network_state_vlan
- instance_nic_bridged_port_isolation
- instance_bulk_state_change
- network_gvrp
- instance_pool_move
- gpu_sriov
- pci_device_type
- storage_volume_state
- network_acl
- migration_stateful
- disk_state_quota
- storage_ceph_features
- projects_compression
- projects_images_remote_cache_expiry
- certificate_project
- network_ovn_acl
- projects_images_auto_update
- projects_restricted_cluster_target
- images_default_architecture
- network_ovn_acl_defaults
- gpu_mig
- project_usage
- network_bridge_acl
- warnings
- projects_restricted_backups_and_snapshots
- clustering_join_token
- clustering_description
- server_trusted_proxy
- clustering_update_cert
- storage_api_project
- server_instance_driver_operational
- server_supported_storage_drivers
- event_lifecycle_requestor_address
- resources_gpu_usb
- clustering_evacuation
- network_ovn_nat_address
- network_bgp
- network_forward
- custom_volume_refresh
- network_counters_errors_dropped
- metrics
- image_source_project
- clustering_config
- network_peer
- linux_sysctl
- network_dns
- ovn_nic_acceleration
- certificate_self_renewal
- instance_project_move
- storage_volume_project_move
- cloud_init
- network_dns_nat
- database_leader
- instance_all_projects
- clustering_groups
- ceph_rbd_du
- instance_get_full
- qemu_metrics
- gpu_mig_uuid
- event_project
- clustering_evacuation_live
- instance_allow_inconsistent_copy
- network_state_ovn
- storage_volume_api_filtering
- image_restrictions
- storage_zfs_export
- network_dns_records
- storage_zfs_reserve_space
- network_acl_log
- storage_zfs_blocksize
- metrics_cpu_seconds
- instance_snapshot_never
- certificate_token
- instance_nic_routed_neighbor_probe
- event_hub
- agent_nic_config
- projects_restricted_intercept
- metrics_authentication
- images_target_project
- cluster_migration_inconsistent_copy
- cluster_ovn_chassis
- container_syscall_intercept_sched_setscheduler
- storage_lvm_thinpool_metadata_size
api_status: stable
api_version: "1.0"
auth: trusted
public: false
auth_methods:
- tls
environment:
  addresses: []
  architectures:
  - x86_64
  - i686
  certificate: |
    -----BEGIN CERTIFICATE-----
    MIICPjCCAcOgAwIBAgIRAK/45wYrnJBv3W7rl5G8VCAwCgYIKoZIzj0EAwMwRzEc
    MBoGA1UEChMTbGludXhjb250YWluZXJzLm9yZzEnMCUGA1UEAwwecm9vdEBseGQt
    Ynktc291cmNlLWRlYnVnLWphbW15MB4XDTIyMDYxNzEzMTkyOFoXDTMyMDYxNDEz
    MTkyOFowRzEcMBoGA1UEChMTbGludXhjb250YWluZXJzLm9yZzEnMCUGA1UEAwwe
    cm9vdEBseGQtYnktc291cmNlLWRlYnVnLWphbW15MHYwEAYHKoZIzj0CAQYFK4EE
    ACIDYgAEsQfHJMPwgaxJPdKWfVflwJp0zpI41MbJkdFCeAeTVsHtMlwkx7UGrMAo
    DRyFcJYjfG0rDyaQ5hISKgH1S2siOtSisSFjFmfPIwtLAtbUtd08TT91rDFMMkL3
    lTcVz7duo3MwcTAOBgNVHQ8BAf8EBAMCBaAwEwYDVR0lBAwwCgYIKwYBBQUHAwEw
    DAYDVR0TAQH/BAIwADA8BgNVHREENTAzghlseGQtYnktc291cmNlLWRlYnVnLWph
    bW15hwR/AAABhxAAAAAAAAAAAAAAAAAAAAABMAoGCCqGSM49BAMDA2kAMGYCMQCh
    iX/LX1QXAD2+pOAkxMrgoXZcZy7i7vCumxHTYk1HUgF5zYwG1iFL3InVMWGr3awC
    MQD70IQrqABWrwWV1RhPb8jMrVj6HJ37RYunXO9Q+13/Lj7fiBKM29Pwee2/jFV0
    +eM=
    -----END CERTIFICATE-----
  certificate_fingerprint: e9fa0394307ded7be71f61462c9594aa8264dce4e44f5ef52a8d23f0a33cab86
  driver: lxc
  driver_version: 5.0.0~git2209-g5a7b9ce67
  firewall: nftables
  kernel: Linux
  kernel_architecture: x86_64
  kernel_features:
    idmapped_mounts: "true"
    netnsid_getifaddrs: "true"
    seccomp_listener: "true"
    seccomp_listener_continue: "true"
    shiftfs: "true"
    uevent_injection: "true"
    unpriv_fscaps: "true"
  kernel_version: 5.15.0-39-generic
  lxc_features:
    cgroup2: "false"
    core_scheduling: "false"
    devpts_fd: "false"
    idmapped_mounts_v2: "false"
    mount_injection_file: "false"
    network_gateway_device_route: "false"
    network_ipvlan: "false"
    network_l2proxy: "false"
    network_phys_macvlan_mtu: "false"
    network_veth_router: "false"
    pidfd: "false"
    seccomp_allow_deny_syntax: "false"
    seccomp_notify: "false"
    seccomp_proxy_send_notify_fd: "false"
  os_name: Ubuntu
  os_version: "22.04"
  project: default
  server: lxd
  server_clustered: false
  server_event_mode: full-mesh
  server_name: lxd-by-source-debug-jammy
  server_pid: 860
  server_version: 5.0.0
  storage: zfs
  storage_version: 2.1.2-1ubuntu3
  storage_supported_drivers:
  - name: btrfs
    version: 5.16.2
    remote: false
  - name: dir
    version: "1"
    remote: false
  - name: lvm
    version: 2.03.11(2) (2021-01-08) / 1.02.175 (2021-01-08) / 4.45.0
    remote: false
  - name: zfs
    version: 2.1.2-1ubuntu3
    remote: false
```

</details>

# Issue description

Tried launching a ubuntu:jammy container on a fresh ubuntu jammy VM with a freshly compiled LXD 5.0.0

# Steps to reproduce

 Always as root, system is up-to-date.
 
 1. Removed snap's lxd, snap itself and apparmor (just in case - same issue with it) and rebooted the host: `snap remove lxd`, `apt-get purge snapd`, `apt-get purge apparmor`
 1. Downloaded LXD 5.0.0 from https://linuxcontainers.org/downloads/lxd/lxd-5.0.0.tar.gz and unpacked it
 1. Compiled LXD according to instructions: https://github.com/lxc/lxd/blob/lxd-5.0.0/doc/installing.md#installing-lxd-from-source
 1. Wrote the exports from `make deps` to .bashrc and relogged in
 1. Installed `zfsutils-linux` so I can have zfs as storage backend
 1. Started lxd with `$(go env GOPATH)/bin/lxd --debug`
 1. In a separate terminal initialized lxd with `$(go env GOPATH)/bin/lxd init`, everything default, so zfs is selected as backend
 1. And I launched a container with `$(go env GOPATH)/bin/lxc launch ubuntu:jammy jammy`
   - Received: `Error: The instance cannot be started as in Error status`

```
root@lxd-by-source-debug-jammy:~# $(go env GOPATH)/bin/lxd init
Would you like to use LXD clustering? (yes/no) [default=no]: 
Do you want to configure a new storage pool? (yes/no) [default=yes]: 
Name of the new storage pool [default=default]: 
Name of the storage backend to use (lvm, zfs, btrfs, dir) [default=zfs]: 
Create a new ZFS pool? (yes/no) [default=yes]: 
Would you like to use an existing empty block device (e.g. a disk or partition)? (yes/no) [default=no]: 
Size in GB of the new loop device (1GB minimum) [default=5GB]: 
Would you like to connect to a MAAS server? (yes/no) [default=no]: 
Would you like to create a new local network bridge? (yes/no) [default=yes]: 
What should the new bridge be called? [default=lxdbr0]: 
What IPv4 address should be used? (CIDR subnet notation, “auto” or “none”) [default=auto]: 
What IPv6 address should be used? (CIDR subnet notation, “auto” or “none”) [default=auto]: 
Would you like the LXD server to be available over the network? (yes/no) [default=no]: 
Would you like stale cached images to be updated automatically? (yes/no) [default=yes]: 
Would you like a YAML "lxd init" preseed to be printed? (yes/no) [default=no]: 
root@lxd-by-source-debug-jammy:~# $(go env GOPATH)/bin/lxc launch ubuntu:jammy jammy
Creating jammy
Starting jammy                                
Error: The instance cannot be started as in Error status
```

Side note: The instance can also not be deleted anymore:
```
root@lxd-by-source-debug-jammy:~# $(go env GOPATH)/bin/lxc list
+-------+-------+------+------+-----------+-----------+
| NAME  | STATE | IPV4 | IPV6 |   TYPE    | SNAPSHOTS |
+-------+-------+------+------+-----------+-----------+
| jammy | ERROR |      |      | CONTAINER | 0         |
+-------+-------+------+------+-----------+-----------+
root@lxd-by-source-debug-jammy:~# $(go env GOPATH)/bin/lxc delete jammy
Error: The instance is currently running, stop it first or pass --force
root@lxd-by-source-debug-jammy:~# $(go env GOPATH)/bin/lxc delete jammy -f
Error: Stopping the instance failed: The instance is already stopped
```

# Information to attach

 - [x] Any relevant kernel output (`dmesg`)
```
...
[    5.501615] systemd[1]: Starting Create Static Device Nodes in /dev...
[    5.506877] systemd[1]: Finished Apply Kernel Variables.
[    5.544800] systemd[1]: Finished Create Static Device Nodes in /dev.
[    5.553577] systemd[1]: Starting Rule-based Manager for Device Events and Files...
[    5.555494] systemd[1]: Started Journal Service.
[    5.581206] systemd-journald[340]: Received client request to flush runtime journal.
[    6.840377] spl: loading out-of-tree module taints kernel.
[    6.850928] icp: module license 'CDDL' taints kernel.
[    6.850940] Disabling lock debugging due to kernel taint
[    7.026256] ZFS: Loaded module v2.1.2-1ubuntu3, ZFS pool version 5000, ZFS filesystem version 5
[   61.337836] NET: Registered PF_VSOCK protocol family
[   74.824093] bridge: filtering via arp/ip/ip6tables is no longer available by default. Update your scripts to load br_netfilter if you need this.
```
 - [x] Container log: There are no container specific log files:
 
```
root@lxd-by-source-debug-jammy:~# $(go env GOPATH)/bin/lxc info --show-log jammy
Name: jammy
Status: ERROR
Type: container
Architecture: x86_64
Created: 2022/06/17 14:15 UTC
Error: open /var/log/lxd/jammy/lxc.log: no such file or directory
root@lxd-by-source-debug-jammy:~# ls -lah /var/log/lxd/
total 8.0K
drwx------ 2 root root   4.0K Jun 17 13:01 .
drwxrwxr-x 9 root syslog 4.0K Jun 17 13:18 ..
-rw-r--r-- 1 root root      0 Jun 17 13:20 dnsmasq.lxdbr0.log
 ```
 - [x] Container configuration (`lxc config show NAME --expanded`)
```
root@lxd-by-source-debug-jammy:~# $(go env GOPATH)/bin/lxc config show jammy --expanded
architecture: x86_64
config:
  image.architecture: amd64
  image.description: ubuntu 22.04 LTS amd64 (release) (20220609)
  image.label: release
  image.os: ubuntu
  image.release: jammy
  image.serial: "20220609"
  image.type: squashfs
  image.version: "22.04"
  volatile.apply_template: create
  volatile.base_image: f3fedd40fcca261890320ec382cae35b4a78222bbc31a7663e20f2ea7abd4ba0
  volatile.cloud-init.instance-id: 5b1323d2-548c-4873-92c4-505dd41326cc
  volatile.eth0.hwaddr: 00:16:3e:da:4f:4c
  volatile.idmap.base: "0"
  volatile.idmap.next: '[{"Isuid":true,"Isgid":false,"Hostid":1000000,"Nsid":0,"Maprange":1000000000},{"Isuid":false,"Isgid":true,"Hostid":1000000,"Nsid":0,"Maprange":1000000000}]'
  volatile.last_state.idmap: '[]'
  volatile.uuid: 8e120bab-f90f-46b3-86eb-11250f0f2801
devices:
  eth0:
    name: eth0
    network: lxdbr0
    type: nic
  root:
    path: /
    pool: default
    type: disk
ephemeral: false
profiles:
- default
stateful: false
description: ""
```
 - [x] Main daemon log with --debug ~~(at /var/log/lxd/lxd.log or /var/snap/lxd/common/lxd/logs/lxd.log)~~ from stdout/err
   - https://pastebin.ubuntu.com/p/d82h63bWkR/
   - _(There was an error creating your Issue: body is too long (maximum is 65536 characters))_
 - [ ] Output of the client with --debug
 - [ ] Output of the daemon with --debug (alternatively output of `lxc monitor` while reproducing the issue)
 - [x] Image unpacked in zfs:

 ```
 root@lxd-by-source-debug-jammy:~# ls -lah /var/lib/lxd/storage-pools/default/containers/jammy
total 8.0K
d--x------ 2 root root 4.0K Jun 17 14:15 .
drwx--x--x 4 root root 4.0K Jun 17 14:22 ..
root@lxd-by-source-debug-jammy:~# mount -t zfs default/containers/jammy /var/lib/lxd/storage-pools/default/containers/jammy
root@lxd-by-source-debug-jammy:~# ls -lah /mnt/
total 8.0K
drwxr-xr-x  2 root root 4.0K Apr 23 02:08 .
drwxr-xr-x 18 root root 4.0K Jun 17 14:13 ..
root@lxd-by-source-debug-jammy:~# ls -lah /var/lib/lxd/storage-pools/default/containers/jammy
total 11K
d--x------  4 root root    6 Jun 17 14:15 .
drwx--x--x  4 root root 4.0K Jun 17 14:22 ..
-r--------  1 root root 2.4K Jun 17 14:15 backup.yaml
-rw-r--r--  1 root root  295 Jun  9 02:44 metadata.yaml
drwxr-xr-x 18 root root   24 Jun  9 02:14 rootfs
drwxr-xr-x  2 root root    3 Jun  9 02:44 templates
```

**Comment by tomponline** [comment]
Your `lxc info` shows that LXD hasn't detected the presence of QEMU I think.

Expect to see something like:

```
  driver: lxc | qemu
  driver_version: 5.0.0 | 6.2.93
```

**Comment by dionysius** [comment]
I thought qemu is only needed for VM's. I tried to launch a container. Sorry if my title looks misleading.

**Comment by tomponline** [comment]
> I thought qemu is only needed for VM's. I tried to launch a container. Sorry if my title looks misleading.

Oh right yes it mislead me.

There's the problem:

```
Error: open /var/log/lxd/jammy/lxc.log: no such file or directory
```

**Comment by dionysius** [comment]
Jes, that means we have no lxc config there as well. But I can't read/find anything in the lxd debug log that gives me a hint why LXD can't write to that folder. It is obviously missing in the beginning (/var/log/lxd exists and gets created by LXD)

**Comment by dionysius** [comment]
The operation for starting the container just says "Error" and that's it.

**Comment by tomponline** [comment]
"The instance cannot be started as in Error status" suggests it cannot ascertain what status the container is in, suggesting there is an issue with LXD talking to liblxc. 

If you delete the container and instead do `lxc init ...` what does `lxc ls` show then?

**Comment by dionysius** [comment]
Deletion also fails, see section "Side note: The instance can also not be deleted anymore:"
> ```
> root@lxd-by-source-debug-jammy:~# $(go env GOPATH)/bin/lxc list
> +-------+-------+------+------+-----------+-----------+
> | NAME  | STATE | IPV4 | IPV6 |   TYPE    | SNAPSHOTS |
> +-------+-------+------+------+-----------+-----------+
> | jammy | ERROR |      |      | CONTAINER | 0         |
> +-------+-------+------+------+-----------+-----------+
> root@lxd-by-source-debug-jammy:~# $(go env GOPATH)/bin/lxc delete jammy
> Error: The instance is currently running, stop it first or pass --force
> root@lxd-by-source-debug-jammy:~# $(go env GOPATH)/bin/lxc delete jammy -f
> Error: Stopping the instance failed: The instance is already stopped
```

so I will create just a new one for now:
```
root@lxd-by-source-debug-jammy:~# $(go env GOPATH)/bin/lxc init ubuntu:jammy jammy3
Creating jammy3
Error: Failed instance creation: Failed creating instance record: Failed initialising instance: Invalid devices: Failed detecting root disk device: No root device could be found
```

Ha! Here we go! My mistake. I started from fresh again and it looks now like this:

```
root@lxd-by-source-debug-jammy:~# $(go env GOPATH)/bin/lxc profile device add default root disk pool=default path=/
Device root added to default
root@lxd-by-source-debug-jammy:~# $(go env GOPATH)/bin/lxc profile show default
config: {}
description: Default LXD profile
devices:
  eth0:
    name: eth0
    nictype: bridged
    parent: lxdbr0
    type: nic
  root:
    path: /
    pool: default
    type: disk
name: default
used_by: []
root@lxd-by-source-debug-jammy:~# $(go env GOPATH)/bin/lxc storage show default
config:
  size: 15GB
  source: /var/lib/lxd/disks/default.img
  zfs.pool_name: default
description: ""
name: default
driver: zfs
used_by:
- /1.0/profiles/default
status: Created
locations:
- none
```

Trying again with init:
```
root@lxd-by-source-debug-jammy:~# $(go env GOPATH)/bin/lxc init ubuntu:jammy jammy3
Creating jammy3
root@lxd-by-source-debug-jammy:~# $(go env GOPATH)/bin/lxc ls
+--------+-------+------+------+-----------+-----------+
|  NAME  | STATE | IPV4 | IPV6 |   TYPE    | SNAPSHOTS |
+--------+-------+------+------+-----------+-----------+
| jammy  | ERROR |      |      | CONTAINER | 0         |
+--------+-------+------+------+-----------+-----------+
| jammy2 | ERROR |      |      | CONTAINER | 0         |
+--------+-------+------+------+-----------+-----------+
| jammy3 | ERROR |      |      | CONTAINER | 0         |
+--------+-------+------+------+-----------+-----------+
```

But still an error. But then the error happens before starting

**Comment by dionysius** [comment]
lxd --debug output only for `lxc init ubuntu:jammy jammy4`:

```
DEBUG  [2022-06-17T15:21:28Z] Handling API request                          ip=@ method=GET protocol=unix url=/1.0 username=root
DEBUG  [2022-06-17T15:21:28Z] WriteJSON
	{
		"type": "sync",
		"status": "Success",
		"status_code": 200,
		"operation": "",
		"error_code": 0,
		"error": "",
		"metadata": {
			"config": {},
			"api_extensions": [
				"storage_zfs_remove_snapshots",
				"container_host_shutdown_timeout",
				"container_stop_priority",
				"container_syscall_filtering",
				"auth_pki",
				"container_last_used_at",
				"etag",
				"patch",
				"usb_devices",
				"https_allowed_credentials",
				"image_compression_algorithm",
				"directory_manipulation",
				"container_cpu_time",
				"storage_zfs_use_refquota",
				"storage_lvm_mount_options",
				"network",
				"profile_usedby",
				"container_push",
				"container_exec_recording",
				"certificate_update",
				"container_exec_signal_handling",
				"gpu_devices",
				"container_image_properties",
				"migration_progress",
				"id_map",
				"network_firewall_filtering",
				"network_routes",
				"storage",
				"file_delete",
				"file_append",
				"network_dhcp_expiry",
				"storage_lvm_vg_rename",
				"storage_lvm_thinpool_rename",
				"network_vlan",
				"image_create_aliases",
				"container_stateless_copy",
				"container_only_migration",
				"storage_zfs_clone_copy",
				"unix_device_rename",
				"storage_lvm_use_thinpool",
				"storage_rsync_bwlimit",
				"network_vxlan_interface",
				"storage_btrfs_mount_options",
				"entity_description",
				"image_force_refresh",
				"storage_lvm_lv_resizing",
				"id_map_base",
				"file_symlinks",
				"container_push_target",
				"network_vlan_physical",
				"storage_images_delete",
				"container_edit_metadata",
				"container_snapshot_stateful_migration",
				"storage_driver_ceph",
				"storage_ceph_user_name",
				"resource_limits",
				"storage_volatile_initial_source",
				"storage_ceph_force_osd_reuse",
				"storage_block_filesystem_btrfs",
				"resources",
				"kernel_limits",
				"storage_api_volume_rename",
				"macaroon_authentication",
				"network_sriov",
				"console",
				"restrict_devlxd",
				"migration_pre_copy",
				"infiniband",
				"maas_network",
				"devlxd_events",
				"proxy",
				"network_dhcp_gateway",
				"file_get_symlink",
				"network_leases",
				"unix_device_hotplug",
				"storage_api_local_volume_handling",
				"operation_description",
				"clustering",
				"event_lifecycle",
				"storage_api_remote_volume_handling",
				"nvidia_runtime",
				"container_mount_propagation",
				"container_backup",
				"devlxd_images",
				"container_local_cross_pool_handling",
				"proxy_unix",
				"proxy_udp",
				"clustering_join",
				"proxy_tcp_udp_multi_port_handling",
				"network_state",
				"proxy_unix_dac_properties",
				"container_protection_delete",
				"unix_priv_drop",
				"pprof_http",
				"proxy_haproxy_protocol",
				"network_hwaddr",
				"proxy_nat",
				"network_nat_order",
				"container_full",
				"candid_authentication",
				"backup_compression",
				"candid_config",
				"nvidia_runtime_config",
				"storage_api_volume_snapshots",
				"storage_unmapped",
				"projects",
				"candid_config_key",
				"network_vxlan_ttl",
				"container_incremental_copy",
				"usb_optional_vendorid",
				"snapshot_scheduling",
				"snapshot_schedule_aliases",
				"container_copy_project",
				"clustering_server_address",
				"clustering_image_replication",
				"container_protection_shift",
				"snapshot_expiry",
				"container_backup_override_pool",
				"snapshot_expiry_creation",
				"network_leases_location",
				"resources_cpu_socket",
				"resources_gpu",
				"resources_numa",
				"kernel_features",
				"id_map_current",
				"event_location",
				"storage_api_remote_volume_snapshots",
				"network_nat_address",
				"container_nic_routes",
				"rbac",
				"cluster_internal_copy",
				"seccomp_notify",
				"lxc_features",
				"container_nic_ipvlan",
				"network_vlan_sriov",
				"storage_cephfs",
				"container_nic_ipfilter",
				"resources_v2",
				"container_exec_user_group_cwd",
				"container_syscall_intercept",
				"container_disk_shift",
				"storage_shifted",
				"resources_infiniband",
				"daemon_storage",
				"instances",
				"image_types",
				"resources_disk_sata",
				"clustering_roles",
				"images_expiry",
				"resources_network_firmware",
				"backup_compression_algorithm",
				"ceph_data_pool_name",
				"container_syscall_intercept_mount",
				"compression_squashfs",
				"container_raw_mount",
				"container_nic_routed",
				"container_syscall_intercept_mount_fuse",
				"container_disk_ceph",
				"virtual-machines",
				"image_profiles",
				"clustering_architecture",
				"resources_disk_id",
				"storage_lvm_stripes",
				"vm_boot_priority",
				"unix_hotplug_devices",
				"api_filtering",
				"instance_nic_network",
				"clustering_sizing",
				"firewall_driver",
				"projects_limits",
				"container_syscall_intercept_hugetlbfs",
				"limits_hugepages",
				"container_nic_routed_gateway",
				"projects_restrictions",
				"custom_volume_snapshot_expiry",
				"volume_snapshot_scheduling",
				"trust_ca_certificates",
				"snapshot_disk_usage",
				"clustering_edit_roles",
				"container_nic_routed_host_address",
				"container_nic_ipvlan_gateway",
				"resources_usb_pci",
				"resources_cpu_threads_numa",
				"resources_cpu_core_die",
				"api_os",
				"container_nic_routed_host_table",
				"container_nic_ipvlan_host_table",
				"container_nic_ipvlan_mode",
				"resources_system",
				"images_push_relay",
				"network_dns_search",
				"container_nic_routed_limits",
				"instance_nic_bridged_vlan",
				"network_state_bond_bridge",
				"usedby_consistency",
				"custom_block_volumes",
				"clustering_failure_domains",
				"resources_gpu_mdev",
				"console_vga_type",
				"projects_limits_disk",
				"network_type_macvlan",
				"network_type_sriov",
				"container_syscall_intercept_bpf_devices",
				"network_type_ovn",
				"projects_networks",
				"projects_networks_restricted_uplinks",
				"custom_volume_backup",
				"backup_override_name",
				"storage_rsync_compression",
				"network_type_physical",
				"network_ovn_external_subnets",
				"network_ovn_nat",
				"network_ovn_external_routes_remove",
				"tpm_device_type",
				"storage_zfs_clone_copy_rebase",
				"gpu_mdev",
				"resources_pci_iommu",
				"resources_network_usb",
				"resources_disk_address",
				"network_physical_ovn_ingress_mode",
				"network_ovn_dhcp",
				"network_physical_routes_anycast",
				"projects_limits_instances",
				"network_state_vlan",
				"instance_nic_bridged_port_isolation",
				"instance_bulk_state_change",
				"network_gvrp",
				"instance_pool_move",
				"gpu_sriov",
				"pci_device_type",
				"storage_volume_state",
				"network_acl",
				"migration_stateful",
				"disk_state_quota",
				"storage_ceph_features",
				"projects_compression",
				"projects_images_remote_cache_expiry",
				"certificate_project",
				"network_ovn_acl",
				"projects_images_auto_update",
				"projects_restricted_cluster_target",
				"images_default_architecture",
				"network_ovn_acl_defaults",
				"gpu_mig",
				"project_usage",
				"network_bridge_acl",
				"warnings",
				"projects_restricted_backups_and_snapshots",
				"clustering_join_token",
				"clustering_description",
				"server_trusted_proxy",
				"clustering_update_cert",
				"storage_api_project",
				"server_instance_driver_operational",
				"server_supported_storage_drivers",
				"event_lifecycle_requestor_address",
				"resources_gpu_usb",
				"clustering_evacuation",
				"network_ovn_nat_address",
				"network_bgp",
				"network_forward",
				"custom_volume_refresh",
				"network_counters_errors_dropped",
				"metrics",
				"image_source_project",
				"clustering_config",
				"network_peer",
				"linux_sysctl",
				"network_dns",
				"ovn_nic_acceleration",
				"certificate_self_renewal",
				"instance_project_move",
				"storage_volume_project_move",
				"cloud_init",
				"network_dns_nat",
				"database_leader",
				"instance_all_projects",
				"clustering_groups",
				"ceph_rbd_du",
				"instance_get_full",
				"qemu_metrics",
				"gpu_mig_uuid",
				"event_project",
				"clustering_evacuation_live",
				"instance_allow_inconsistent_copy",
				"network_state_ovn",
				"storage_volume_api_filtering",
				"image_restrictions",
				"storage_zfs_export",
				"network_dns_records",
				"storage_zfs_reserve_space",
				"network_acl_log",
				"storage_zfs_blocksize",
				"metrics_cpu_seconds",
				"instance_snapshot_never",
				"certificate_token",
				"instance_nic_routed_neighbor_probe",
				"event_hub",
				"agent_nic_config",
				"projects_restricted_intercept",
				"metrics_authentication",
				"images_target_project",
				"cluster_migration_inconsistent_copy",
				"cluster_ovn_chassis",
				"container_syscall_intercept_sched_setscheduler",
				"storage_lvm_thinpool_metadata_size"
			],
			"api_status": "stable",
			"api_version": "1.0",
			"auth": "trusted",
			"public": false,
			"auth_methods": [
				"tls"
			],
			"environment": {
				"addresses": [],
				"architectures": [
					"x86_64",
					"i686"
				],
				"certificate": "-----BEGIN CERTIFICATE-----\nMIICPjCCAcOgAwIBAgIRAIbBAvbz+gmTLCHaHiuWHlUwCgYIKoZIzj0EAwMwRzEc\nMBoGA1UEChMTbGludXhjb250YWluZXJzLm9yZzEnMCUGA1UEAwwecm9vdEBseGQt\nYnktc291cmNlLWRlYnVnLWphbW15MB4XDTIyMDYxNzE1MTM1OVoXDTMyMDYxNDE1\nMTM1OVowRzEcMBoGA1UEChMTbGludXhjb250YWluZXJzLm9yZzEnMCUGA1UEAwwe\ncm9vdEBseGQtYnktc291cmNlLWRlYnVnLWphbW15MHYwEAYHKoZIzj0CAQYFK4EE\nACIDYgAEHZxz6xmlwSlM4kUF+4AbF8ly/i5sJQcpQOqhO8nfzqJEDpLMxRmxFqFC\nejf6GQR/sjdZqK2tbQCREtnykJ57H/EfN2g6ujpaoRb+oa+2dDsM/hjOheJ51I63\nUurDpUVgo3MwcTAOBgNVHQ8BAf8EBAMCBaAwEwYDVR0lBAwwCgYIKwYBBQUHAwEw\nDAYDVR0TAQH/BAIwADA8BgNVHREENTAzghlseGQtYnktc291cmNlLWRlYnVnLWph\nbW15hwR/AAABhxAAAAAAAAAAAAAAAAAAAAABMAoGCCqGSM49BAMDA2kAMGYCMQD7\ngThc9cyCv2KLYWhY/16a2ICZ6RWEVB+5hVoac4aDmHWpPfUJFE6xt2x63XxOmDMC\nMQCYYefzVtwEUcWEas4Ae1K2lmnlZf3rH9gif4C0nBIDyVrFK9oYOuaVYQLD8srp\nYD8=\n-----END CERTIFICATE-----\n",
				"certificate_fingerprint": "59c2a5768ddfccb8ecf86aad7563be351bf1b626ddc7a42b69a2121b473bcced",
				"driver": "lxc",
				"driver_version": "5.0.0~git2209-g5a7b9ce67",
				"firewall": "nftables",
				"kernel": "Linux",
				"kernel_architecture": "x86_64",
				"kernel_features": {
					"idmapped_mounts": "true",
					"netnsid_getifaddrs": "true",
					"seccomp_listener": "true",
					"seccomp_listener_continue": "true",
					"shiftfs": "true",
					"uevent_injection": "true",
					"unpriv_fscaps": "true"
				},
				"kernel_version": "5.15.0-39-generic",
				"lxc_features": {
					"cgroup2": "false",
					"core_scheduling": "false",
					"devpts_fd": "false",
					"idmapped_mounts_v2": "false",
					"mount_injection_file": "false",
					"network_gateway_device_route": "false",
					"network_ipvlan": "false",
					"network_l2proxy": "false",
					"network_phys_macvlan_mtu": "false",
					"network_veth_router": "false",
					"pidfd": "false",
					"seccomp_allow_deny_syntax": "false",
					"seccomp_notify": "false",
					"seccomp_proxy_send_notify_fd": "false"
				},
				"os_name": "Ubuntu",
				"os_version": "22.04",
				"project": "default",
				"server": "lxd",
				"server_clustered": false,
				"server_event_mode": "full-mesh",
				"server_name": "lxd-by-source-debug-jammy",
				"server_pid": 13346,
				"server_version": "5.0.0",
				"storage": "zfs",
				"storage_version": "2.1.2-1ubuntu3",
				"storage_supported_drivers": [
					{
						"Name": "btrfs",
						"Version": "5.16.2",
						"Remote": false
					},
					{
						"Name": "dir",
						"Version": "1",
						"Remote": false
					},
					{
						"Name": "lvm",
						"Version": "2.03.11(2) (2021-01-08) / 1.02.175 (2021-01-08) / 4.45.0",
						"Remote": false
					},
					{
						"Name": "zfs",
						"Version": "2.1.2-1ubuntu3",
						"Remote": false
					}
				]
			}
		}
	}  http_code=200
DEBUG  [2022-06-17T15:21:29Z] Handling API request                          ip=@ method=GET protocol=unix url=/1.0/events username=root
DEBUG  [2022-06-17T15:21:29Z] Event listener server handler started         listener=fceadad8-4fed-43b5-b906-262356818425 local=/var/lib/lxd/unix.socket remote=@
DEBUG  [2022-06-17T15:21:29Z] Handling API request                          ip=@ method=POST protocol=unix url=/1.0/instances username=root
DEBUG  [2022-06-17T15:21:29Z] API Request
	{
		"architecture": "",
		"config": {},
		"devices": {},
		"ephemeral": false,
		"profiles": null,
		"stateful": false,
		"description": "",
		"name": "jammy4",
		"source": {
			"type": "image",
			"certificate": "",
			"alias": "jammy",
			"server": "https://cloud-images.ubuntu.com/releases",
			"protocol": "simplestreams",
			"mode": "pull",
			"allow_inconsistent": false
		},
		"instance_type": "",
		"type": "container"
	}  ip=@ method=POST protocol=unix url=/1.0/instances username=root
DEBUG  [2022-06-17T15:21:29Z] Responding to instance create                
DEBUG  [2022-06-17T15:21:29Z] New task Operation: 124248b1-9079-4548-9b9d-6edef2f9f11b 
DEBUG  [2022-06-17T15:21:29Z] Started task operation: 124248b1-9079-4548-9b9d-6edef2f9f11b 
DEBUG  [2022-06-17T15:21:29Z] WriteJSON
	{
		"type": "async",
		"status": "Operation created",
		"status_code": 100,
		"operation": "/1.0/operations/124248b1-9079-4548-9b9d-6edef2f9f11b",
		"error_code": 0,
		"error": "",
		"metadata": {
			"id": "124248b1-9079-4548-9b9d-6edef2f9f11b",
			"class": "task",
			"description": "Creating instance",
			"created_at": "2022-06-17T15:21:29.022476976Z",
			"updated_at": "2022-06-17T15:21:29.022476976Z",
			"status": "Running",
			"status_code": 103,
			"resources": {
				"containers": [
					"/1.0/containers/jammy4"
				],
				"instances": [
					"/1.0/instances/jammy4"
				]
			},
			"metadata": null,
			"may_cancel": false,
			"err": "",
			"location": "none"
		}
	}  http_code=202
DEBUG  [2022-06-17T15:21:29Z] Handling API request                          ip=@ method=GET protocol=unix url=/1.0/operations/124248b1-9079-4548-9b9d-6edef2f9f11b username=root
DEBUG  [2022-06-17T15:21:29Z] Connecting to a remote simplestreams server   URL="https://cloud-images.ubuntu.com/releases"
DEBUG  [2022-06-17T15:21:29Z] WriteJSON
	{
		"type": "sync",
		"status": "Success",
		"status_code": 200,
		"operation": "",
		"error_code": 0,
		"error": "",
		"metadata": {
			"id": "124248b1-9079-4548-9b9d-6edef2f9f11b",
			"class": "task",
			"description": "Creating instance",
			"created_at": "2022-06-17T15:21:29.022476976Z",
			"updated_at": "2022-06-17T15:21:29.022476976Z",
			"status": "Running",
			"status_code": 103,
			"resources": {
				"containers": [
					"/1.0/containers/jammy4"
				],
				"instances": [
					"/1.0/instances/jammy4"
				]
			},
			"metadata": null,
			"may_cancel": false,
			"err": "",
			"location": "none"
		}
	}  http_code=200
DEBUG  [2022-06-17T15:21:29Z] Acquiring lock for image download of "f3fedd40fcca261890320ec382cae35b4a78222bbc31a7663e20f2ea7abd4ba0" 
DEBUG  [2022-06-17T15:21:29Z] Lock acquired for image download of "f3fedd40fcca261890320ec382cae35b4a78222bbc31a7663e20f2ea7abd4ba0" 
DEBUG  [2022-06-17T15:21:29Z] Image already exists in the DB                fingerprint=f3fedd40fcca261890320ec382cae35b4a78222bbc31a7663e20f2ea7abd4ba0
DEBUG  [2022-06-17T15:21:29Z] Acquiring lock for image download of "f3fedd40fcca261890320ec382cae35b4a78222bbc31a7663e20f2ea7abd4ba0" 
DEBUG  [2022-06-17T15:21:29Z] Lock acquired for image download of "f3fedd40fcca261890320ec382cae35b4a78222bbc31a7663e20f2ea7abd4ba0" 
DEBUG  [2022-06-17T15:21:29Z] Instance operation lock created               action=create instance=jammy4 project=default reusable=false
INFO   [2022-06-17T15:21:29Z] Creating container                            ephemeral=false instance=jammy4 instanceType=container project=default
DEBUG  [2022-06-17T15:21:29Z] Database error                                err="Network not found"
DEBUG  [2022-06-17T15:21:29Z] Database error                                err="Network not found"
DEBUG  [2022-06-17T15:21:29Z] Adding device                                 device=eth0 type=nic
DEBUG  [2022-06-17T15:21:29Z] Adding device                                 device=root type=disk
INFO   [2022-06-17T15:21:29Z] Created container                             ephemeral=false instance=jammy4 instanceType=container project=default
DEBUG  [2022-06-17T15:21:29Z] CreateInstanceFromImage started               instance=jammy4 project=default
DEBUG  [2022-06-17T15:21:29Z] EnsureImage started                           fingerprint=f3fedd40fcca261890320ec382cae35b4a78222bbc31a7663e20f2ea7abd4ba0
DEBUG  [2022-06-17T15:21:29Z] Checking image volume size                    fingerprint=f3fedd40fcca261890320ec382cae35b4a78222bbc31a7663e20f2ea7abd4ba0
DEBUG  [2022-06-17T15:21:29Z] Setting image volume size                     fingerprint=f3fedd40fcca261890320ec382cae35b4a78222bbc31a7663e20f2ea7abd4ba0 size=
DEBUG  [2022-06-17T15:21:29Z] EnsureImage finished                          fingerprint=f3fedd40fcca261890320ec382cae35b4a78222bbc31a7663e20f2ea7abd4ba0
DEBUG  [2022-06-17T15:21:29Z] Checking volume size                          instance=jammy4 project=default
DEBUG  [2022-06-17T15:21:29Z] Set new volume size                           instance=jammy4 project=default size=
DEBUG  [2022-06-17T15:21:29Z] Mounted ZFS dataset                           dev=default/containers/jammy4 driver=zfs path=/var/lib/lxd/storage-pools/default/containers/jammy4 pool=default
DEBUG  [2022-06-17T15:21:29Z] Unmounted ZFS dataset                         dev=default/containers/jammy4 driver=zfs path=/var/lib/lxd/storage-pools/default/containers/jammy4 pool=default volName=jammy4
DEBUG  [2022-06-17T15:21:30Z] CreateInstanceFromImage finished              instance=jammy4 project=default
DEBUG  [2022-06-17T15:21:30Z] UpdateInstanceBackupFile started              instance=jammy4 project=default
DEBUG  [2022-06-17T15:21:30Z] Mounted ZFS dataset                           dev=default/containers/jammy4 driver=zfs path=/var/lib/lxd/storage-pools/default/containers/jammy4 pool=default
DEBUG  [2022-06-17T15:21:30Z] Unmounted ZFS dataset                         dev=default/containers/jammy4 driver=zfs path=/var/lib/lxd/storage-pools/default/containers/jammy4 pool=default volName=jammy4
DEBUG  [2022-06-17T15:21:30Z] UpdateInstanceBackupFile finished             instance=jammy4 project=default
DEBUG  [2022-06-17T15:21:30Z] Instance operation lock finished              action=create err="<nil>" instance=jammy4 project=default reusable=false
DEBUG  [2022-06-17T15:21:30Z] Success for task operation: 124248b1-9079-4548-9b9d-6edef2f9f11b 
DEBUG  [2022-06-17T15:21:30Z] Handling API request                          ip=@ method=GET protocol=unix url=/1.0/instances/jammy4 username=root
DEBUG  [2022-06-17T15:21:30Z] WriteJSON
	{
		"type": "sync",
		"status": "Success",
		"status_code": 200,
		"operation": "",
		"error_code": 0,
		"error": "",
		"metadata": {
			"architecture": "x86_64",
			"config": {
				"image.architecture": "amd64",
				"image.description": "ubuntu 22.04 LTS amd64 (release) (20220609)",
				"image.label": "release",
				"image.os": "ubuntu",
				"image.release": "jammy",
				"image.serial": "20220609",
				"image.type": "squashfs",
				"image.version": "22.04",
				"volatile.apply_template": "create",
				"volatile.base_image": "f3fedd40fcca261890320ec382cae35b4a78222bbc31a7663e20f2ea7abd4ba0",
				"volatile.cloud-init.instance-id": "1d35effe-dc6e-4de9-8a98-846d55a91520",
				"volatile.eth0.hwaddr": "00:16:3e:80:26:8a",
				"volatile.idmap.base": "0",
				"volatile.idmap.next": "[{\"Isuid\":true,\"Isgid\":false,\"Hostid\":1000000,\"Nsid\":0,\"Maprange\":1000000000},{\"Isuid\":false,\"Isgid\":true,\"Hostid\":1000000,\"Nsid\":0,\"Maprange\":1000000000}]",
				"volatile.last_state.idmap": "[]",
				"volatile.uuid": "065ec26b-49f2-4242-966f-97a960820d5a"
			},
			"devices": {},
			"ephemeral": false,
			"profiles": [
				"default"
			],
			"stateful": false,
			"description": "",
			"created_at": "2022-06-17T15:21:29.523169889Z",
			"expanded_config": {
				"image.architecture": "amd64",
				"image.description": "ubuntu 22.04 LTS amd64 (release) (20220609)",
				"image.label": "release",
				"image.os": "ubuntu",
				"image.release": "jammy",
				"image.serial": "20220609",
				"image.type": "squashfs",
				"image.version": "22.04",
				"volatile.apply_template": "create",
				"volatile.base_image": "f3fedd40fcca261890320ec382cae35b4a78222bbc31a7663e20f2ea7abd4ba0",
				"volatile.cloud-init.instance-id": "1d35effe-dc6e-4de9-8a98-846d55a91520",
				"volatile.eth0.hwaddr": "00:16:3e:80:26:8a",
				"volatile.idmap.base": "0",
				"volatile.idmap.next": "[{\"Isuid\":true,\"Isgid\":false,\"Hostid\":1000000,\"Nsid\":0,\"Maprange\":1000000000},{\"Isuid\":false,\"Isgid\":true,\"Hostid\":1000000,\"Nsid\":0,\"Maprange\":1000000000}]",
				"volatile.last_state.idmap": "[]",
				"volatile.uuid": "065ec26b-49f2-4242-966f-97a960820d5a"
			},
			"expanded_devices": {
				"eth0": {
					"name": "eth0",
					"nictype": "bridged",
					"parent": "lxdbr0",
					"type": "nic"
				},
				"root": {
					"path": "/",
					"pool": "default",
					"type": "disk"
				}
			},
			"name": "jammy4",
			"status": "Error",
			"status_code": 112,
			"last_used_at": "1970-01-01T00:00:00Z",
			"location": "none",
			"type": "container",
			"project": "default"
		}
	}  http_code=200
DEBUG  [2022-06-17T15:21:30Z] Event listener server handler stopped         listener=fceadad8-4fed-43b5-b906-262356818425 local=/var/lib/lxd/unix.socket remote=@
```

**Comment by dionysius** [comment]
Ok, I did everything again with LXD 5.2 and the issue is gone! So you might have fixed it already. But I can retry it once LXD 5.0.1 comes out.

I have another issue with LXD 5.0 I wanted to reproduce but got stuck on this error.

**Comment by stgraber** [comment]
The problem you're running into is LXD not being able to parse the liblxc version number.

We've fixed the shortly after LXD 5.0 and the fix will be in LXD 5.0.1.

We also released the final version of LXC 5.0 yesterday which wouldn't have the version number problem but it may take a while before it makes it to Ubuntu.
